microlith/grafana/provisioning/dashboards/couchbase-inventory.json

```diff
@@ -278,29 +287,29 @@
               "name": "Services"
             },
             {
-              "jsonPath": "$[*].status_summary.good",
+              "jsonPath": "$[*].status_summary.alerts",
               "language": "jsonpath",
-              "name": "Status: Good"
+              "name": "Critical"
             },
             {
               "jsonPath": "$[*].status_summary.warnings",
               "language": "jsonpath",
-              "name": "Status: Warning"
+              "name": "Warnings"
             },
             {
-              "jsonPath": "$[*].status_summary.alerts",
+              "jsonPath": "$[*].status_summary.good",
               "language": "jsonpath",
-              "name": "Status: Alert"
+              "name": "Successful"
             },
             {
               "jsonPath": "$[*].status_summary.info",
               "language": "jsonpath",
-              "name": "Status: Info"
+              "name": "Info Alerts"
             },
             {
               "jsonPath": "$map($, function($_) {\"Cluster Overview\"})",
               "language": "jsonata",
-              "name": "Links"
+              "name": "Dashboard"
             }
```

Comment:
We can get rid of this field entirely if we're using the name for the link, that'll also mean we don't need the transformation that hides it